msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md

```diff
@@ -64,7 +64,7 @@ The **scanBarCode()** API supports the following barcode types:
 
 The following image depicts web app experience of QR or barcode scanner capability:
 
-![web app experience for qr or barcode scanner capability](../../assets/images/tabs/qr-barcode-scanner-capability.png)
+:::image type="content" source="../../assets/images/tabs/qr-barcode-scanner-capability.png" alt-text="The screenshot is an example that shows Web app experience for qr or barcode scanner capability.":::
```

Comment:
What casing should `qr` word have?
The alt text is copied from the previous line above the image and yet this casing is incorrect! What's happening here @v-habibaumme ?

Reply:
resolved